android/src/main/java/com/reactnativecompressor/Utils/createVideoThumbnail.kt

```diff
@@ -6,10 +6,10 @@ import android.graphics.BitmapFactory
 import android.media.MediaMetadataRetriever
 import android.net.Uri
 import android.os.Build
+import android.os.AsyncTask
```

Comment:
as far as I know... AsyncTask is deprecated in Java.
is there any alternative way?